core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -169,6 +185,7 @@ private[log] class LogCleanerManager(val logDirs: Array[File], val logs: Pool[To
           state match {
             case LogCleaningPaused =>
               inProgress.remove(topicAndPartition)
+              decrementNumFilthyLogs()
```

Comment:
Hmm, we probably shouldn't decrement here since this just allows new cleaning to be started.